### ISSUE TITLE
angular 5 support

### ISSUE THREAD

**Issue by ctaepper** [UltimateAngular/ngx-errors#39]
```json
  "peerDependencies": {
    "@angular/forms": "^2.3.1 || >=4.0.0-beta <5.0.0"
  },
```

can you make it ` || ^5.0.0` ?

**Comment by antoantonyk** [comment]
@toddmotto Can you please update the peerDependencies?

**Comment by kotmatpockuh** [comment]
@toddmotto U alive?

any response will do us good...

**Comment by toddmotto** [comment]
@antoantonyk @ctaepper Just published v1.4.0 which adds `^5.0.0` to `peerDependencies` :)
